js-basics/js-functions_functions-with-parameters/js/index.js

```diff
@@ -37,12 +35,18 @@ greetMary();
 
 // --v-- write your code here --v--
 
-function greet() {}
+function greet(name) {
+  console.log("Welcome " + name + ",good to see you again!");
```

Comment:
Tiny note, you can add a space before `,good` then you'll get
`Welcome Bob, good to see you again!`
instead of
`Welcome Bob,good to see you again!`

Reply:
thanx))